Pages/Pag81751.Result.al

```diff
@@ -0,0 +1,63 @@
+/// <summary>
+/// Page to show the result
+/// </summary>
+page 81752 Result
+{
+    ApplicationArea = All;
+    Caption = 'Result';
+    PageType = Worksheet;
```

Comment:
@NAVFreak 
Just wondering, why Worksheet specifically? :)

Reply:
He he, I'm not a "frontend" guy 🤪. I just grabbed the first list that I thought I could use were I wanted to edit 1 field (comments). Sometimes you forgot to switch between runs or wan't to change it afterwards.